docker-compose.dev.yml

```diff
@@ -36,6 +36,7 @@ services:
       DEBUG: '*'
     volumes:
       - ./db:/app/db
+      - ./generated:/app/generated
```

Comment:
is this related to this PR?

Reply:
this is a small fix for the local dev env. Without it dev containers didn't pick up new db functions